c/gonzo.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
```

Comment:
Please leave this file off.

Reply:
Good catch, removed with b430c289e54354d3116a60dfdfac8d487d2b6cc7.